pc2v9.ini

```diff
@@ -25,4 +25,6 @@ server=localhost:50002
 wtiport=8080
 wtiwsName=/websocket
 
+[clics]
+apiVersionsSupported=2023-06,2020-03
```

Comment:
I wonder if the name of this key shouldn't be "clicsApiVersionsSupported"?  Granted, it's in the [clics] section, but it still seems that someone might confuse it with "pc2Api"...

Reply:
I *think* the idea is to phase out the pc2api out altogether.  If the CLICS api interface can do everything the pc2api can, and cleaner (ie restful interface), then we can phase it out.

Reply:
Well, IF the clics API could do everything the pc2api can do, then I would agree that we would probably just deprecate and then remove the pc2api.  However, I think there are MANY things that the pc2api can do that are not now, and would not ever be accepted in the future, as CLICS API features.  For example, with the pc2api you can register listeners for new submissions, new clarifications, contest configuration changes, etc.  I cannot imagine [certain people] ever being willing to include that in the CLICS API.

Having said that, I'm not at all adamant about this particular suggested change; I can live with just calling it `apiVersionsSupported` since it would only be relevant (or accessible) within the `clics` section of the `ini` file.

Reply:
You CAN in fact get all those things you mention on the event feed.  Instead of listeners, you listen to the event feed for submissions, clars, state/contest changes, etc.  So, it's already part of the CLICS api.  This is, in fact what I do with my WJI stuff for clars and state messages...